universal-login-wallet/package.json

```diff
@@ -30,7 +30,8 @@
     "react-dom": "^16.8.6",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "react-transition-group": "^2.8.0"
+    "react-transition-group": "^2.8.0",
+    "react-use-mutable": "^0.1.1"
```

Comment:
No production of micro-npms, they introduce additional dependency management and security risk.